packages/expo-linear-gradient/android/src/main/java/expo/modules/lineargradient/LinearGradientModule.kt

```diff
@@ -0,0 +1,39 @@
+package expo.modules.lineargradient
+
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.module
+
+typealias ViewType = LinearGradientView
+
+class LinearGradientModule : Module() {
+  override fun definition() = module {
+    name("ExpoLinearGradient")
+    viewManager {
+      view { context ->
+        LinearGradientView(context)
+      }
+
+      prop("colors") { view: ViewType, colors: IntArray ->
+        view.setColors(colors)
+      }
+
+      prop("locations") { view: ViewType, locations: FloatArray? ->
+        locations?.let {
+          view.setLocations(it)
+        }
+      }
+
+      prop("startPoint") { view: ViewType, startPoint: Pair<Float, Float>? ->
+        view.setStartPosition(startPoint?.first ?: 0f, startPoint?.second ?: 0f)
```

Comment:
`(0.5, 0.0)` was the default start point on iOS
```suggestion
        view.setStartPosition(startPoint?.first ?: 0.5, startPoint?.second ?: 0f)
```